core/templates/dev/head/pages/exploration_editor/editor_tab/StateSolution.js

```diff
@@ -61,7 +62,9 @@ oppia.controller('StateSolution', [
 
     $scope.getSolutionSummary = function() {
       var solution = stateSolutionService.savedMemento;
-      return solution.getSummary(stateInteractionIdService.savedMemento);
+      var solutionAsPlainText = $filter('convertToPlainText')(
+        solution.explanation._html);
```

Comment:
1. Use getHtml()

2. This doesn't seem to produce the same text as before; we want to show the solution in the abbreviated header as well.